src/test/java/core/basesyntax/PalindromeTest.java

```diff
@@ -6,34 +6,40 @@
 public class PalindromeTest {
     private static final String LETTERS_PALINDROME = "Madam, I'm Adam!";
     private static final String LETTERS_AND_NUMBERS_PALINDROME = "1.) level-1";
+    private static final String LETTERS_AND_NUMBERS_NON_PALINDROME = ".) level-1";
     private static final String NUMBERS_PALINDROME = "123 43 21";
     private static final String NON_PALINDROME = "Nice day!";
 
+
     @Test
     public void isPalindromeWithAllLetters() {
         Palindrome palindrome = new Palindrome();
         boolean actualResult = palindrome.isPalindrome(LETTERS_PALINDROME);
-        Assert.assertTrue("Test faid with text " + LETTERS_PALINDROME, actualResult);
+        Assert.assertTrue("Test failed with text " + LETTERS_PALINDROME, actualResult);
     }
 
     @Test
     public void isPalindromeWithLettersAndNumbers() {
```

Comment:
Test case per scenario please. Looks like you've added new scenario to this test case.